schemas/user_request.py

```diff
@@ -21,12 +21,16 @@ class UserRequestExistenceOut(BaseModel):
 
 
 class UserRequestBookingHostelOut(BaseModel):
-    full_name: str
+    first_name: str
+    last_name: str
+    middle_name: str
     user_id: int
     faculty_name: str
     university_id: int
     short_university_name: str
-    rector_full_name: str
+    rector_first_name: str
```

Comment:
remove unnecessary prefixes
Maybe use the embedded key: "rector" with first, last, and middle names...